pyproject.toml.setuppy

```diff
@@ -5,5 +5,5 @@
 requires = [
     "setuptools==59.2.0",
     "wheel==0.38.1",
-    "Cython>=0.29.34,<3.0",
+    "Cython>=0.29.34,<3.1",
```

Comment:
Do you intend pinning the upper value?

Reply:
This seems okay to me, since this is a fallback file for switching back to `setup.py` based builds via `pip`. For releases we do tend to want upper bounds there.